h/config.py

```diff
@@ -70,9 +70,19 @@
                       message='use the AUTHORITY environment variable instead'),
     EnvSetting('h.authority', 'AUTHORITY'),
     EnvSetting('h.bouncer_url', 'BOUNCER_URL'),
+
+    # ID and secret used for the issuer when signing JWT tokens for legacy API
+    # auth.
     EnvSetting('h.client_id', 'CLIENT_ID'),
     EnvSetting('h.client_secret', 'CLIENT_SECRET'),
+
     EnvSetting('h.client_url', 'CLIENT_URL'),
+
+    # ID for the OAuth authclient that the embedded client should use when
+    # making requests to OAuth endpoints. As a public client, it does not have a
+    # secret.
+    EnvSetting('h.client_oauth_id', 'CLIENT_OAUTH_ID'),
```

Comment:
I wonder if we should add the new env var here? http://h.readthedocs.io/en/latest/developing/envvars/ Of course most h env vars are not documented there but they probably should be